doc/dev/doc/linq.md

```diff
@@ -0,0 +1,49 @@
+﻿# LINQ Provider (BETA)
+Hazelcast .Net Client provides a LINQ provider over `IHMap`. Provider is currently in BETA version.
+You can use programmatic LINQ functions instead string SQL statements to query over your distributed map.
+To benefit from LINQ support, `Hazelcast.Net.Linq.Async` package should be added your dependency. The package is an
+extension of `Hazelcast.Net`. It depends on it. The provider uses Hazelcast .Net Client underneath. Both packages are in NuGet. 
+
+#### Supported LINQ Operations
+
+- Where
+- Select
+
+#### Remarks
+Linq provider translates your expression to SQL statements, and send it to server via SQL Service of the Client. 
+It requires same steps as SQL. The map should be mapped on the server side, and your property names should match 
+with configured column names on mapping. For primitive types `__key` and `this` keywords will be used. For complex types,
+property name will be used as it is. Also, note that properties should be set publicly. Otherwise, the result object cannot be
```

Comment:
```suggestion
property name will be used as it is. Also, note that properties should be publicly-settable. Otherwise, the result object cannot be
```

Reply:
Fixed at https://github.com/hazelcast/hazelcast-csharp-client/pull/759/commits/eedb97b777a60525334a90a76da72e794066f235